BambooTracker/BambooTracker.pro

```diff
@@ -391,4 +390,38 @@ RESOURCES += \
 TRANSLATIONS += \
     res/lang/bamboo_tracker_fr.ts
 
+equals(QT_MAJOR_VERSION, 5) {
+    lessThan(QT_MINOR_VERSION, 12) {
+        message(using a workaround for missing 'lrelease' option in Qt <5.12...)
+        
+        for(tsfile, TRANSLATIONS) {
+          qmfile   = $$tsfile
+          qmfile  ~= s/.ts$/.qm/
+          qmlang   = $$tsfile
+          qmlang  ~= s,res/lang/bamboo_tracker_,,
+          qmlang  ~= s,.ts$,,
+          
+            thisqm = $${qmfile}.commands
+          $$thisqm = lrelease $$tsfile
+          
```

Comment:
I think you should add the TS source as a dependency of QM.
```suggestion
            thisqmdep = $${qmfile}.depends
          $$thisqmdep = $${tsfile}
```

Reply:
Added in https://github.com/rerrahkr/BambooTracker/pull/61/commits/c3d4f318b775afb0e66f43f988ad2a43149fb0f2.